tests/FSharp.Test.Utilities/FSharp.Test.Utilities.fsproj

```diff
@@ -64,6 +67,9 @@
     <!-- CompilerAssert dependencies.
          Make sure they are getting built with the Utilities. -->
     <ProjectReference Include="$(FSharpSourcesRoot)\FSharp.Build\FSharp.Build.fsproj" />
+    <ProjectReference Include="$(FSharpSourcesRoot)\FSharp.Compiler.Interactive.Settings\FSharp.Compiler.Interactive.Settings.fsproj" />
```

Comment:
Are these refs still needed? Can't understand what makes in necessary now, probably missing something.

Reply:
I see :D Well ideally we'd not have them here since this is not the dependency flow we wish to have in the solution. So if possible, please remove them.